pages/artist/_itemId/index.vue

```diff
@@ -5,12 +5,13 @@
         <v-row>
           <v-col cols="9" class="d-flex flex-row">
             <v-img
-              v-if="item.ImageTags && item.ImageTags.Primary"
-              class="person-image elevation-2 ml-2"
+              v-if="item.Id"
               cover
               aspect-ratio="1"
-              :src="getImageUrl(item.Id, 'Primary')"
-            />
+              class="person-image elevation-2 ml-2"
+            >
+              <blurhash-image :item="item" />
+            </v-img>
```

Comment:
Why is `<blurhash-image />` inside a `<v-img>` here?

Reply:
For applying the correct styling easier without us needing to deal with adding additional props to the component.